### ISSUE TITLE
React native Issues

### ISSUE THREAD

**Issue by bakerac4** [true-myth/true-myth#239]
Im seeing a few errors when trying to use this in a react native project that I wanted to report.

1.
```
warn Package true-myth has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /node_modules/true-myth/package.json`.
 This seems to be fixed just by adding  `"./package.json": "./package.json",` in the exports section of the package.json in node_modules. 
```
2. 
```
LogBox.js:173 Require cycle: node_modules/true-myth/dist/result.js -> node_modules/true-myth/dist/maybe.js -> node_modules/true-myth/dist/result.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle.
```

It seems to be working otherwise though.
Node: 16.13.0
NPM: 8.1.0

**Comment by chriskrycho** [comment]
Interesting. The first issue seems totally spurious to me: `package.json` isn't normally something you'd put in `exports`—within `package.json` to boot! What versions of Node and webpack/Parcel/etc. are you using?

The second issue is just an overly opinionated warning and is definitely safe to ignore. Cycles between are perfectly legitimate and these ones in particular don't cause initialization issues: there are no module-scoped operations other than declarations.

**Comment by chriskrycho** [comment]
Update: I just reread all of the docs here, and it looks like we do need to add that. I will publish a bug fix release to make this work for things which explicitly use the `package.json` file.

**Comment by chriskrycho** [comment]
Published a fix in [v5.0.1])(https://github.com/true-myth/true-myth/releases/tag/v5.0.1).
